src/nuget/Microsoft.DotNet.BuildTools.nuspec

```diff
@@ -12,11 +12,12 @@
     <iconUrl>http://go.microsoft.com/fwlink/?LinkID=288859</iconUrl>
     <summary>.NET Core Framework Build Tools</summary>
     <description>This package provides build-time support for the .NET Core Framework libraries.  You should not need to reference this package from your project.</description>
-    <copyright>Copyright © Microsoft Corporation</copyright>
+    <copyright>Copyright � Microsoft Corporation</copyright>
     <tags></tags>
```

Comment:
Don't you want to update the build tools version number as well? Or are you planning to do that at another point?